### DISCUSSION TITLE
Regarding Self learning

### DISCUSSION THREAD

**Discussion by Moulya-Nerella** [github/india#307] [Students]
Hi guys, I have a small issue that i completed a module in self learning but it is not showing complete still it's displaying 'in progress' How can i clear out this?

**Comment by Abhishek-1228** [comment]
same issue plz help!

**Comment by gauravjejurkar** [comment]
i also have same issue

**Comment by saaiaadityaayenduri1** [comment]
Can we learn self learning courses by downloading ??

**Comment by VarunSarkar** [comment]
I am facing the same issue too. Also for the certifications do we have to pay for them in learning section
?

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
